students/donghun/users/views.py

```diff
@@ -32,10 +36,26 @@ def post(self,request):
             User.objects.create(
```

Comment:
`bcrypt.hashpw`의 경우 어차피 정규표현식을 통과해야만 활용되는 코드이기 때문에 정규표현식 검증을 다 통과하고 난 뒤 hash 된 비밀번호를 넣어서 create 하는 것이 더 효율적으로 보입니다.
```suggestion
            hashed_password = bcrypt.hashpw(...).decode('utf-8')
            User.objects.create(
                ...
                password = hashed_password
            )
```